patches/activate-cross-signing-and-secure-storage-react/matrix-react-sdk+3.63.0.patch

```diff
@@ -126,7 +126,87 @@ index 2e6e6fb..dae664a 100644
  
  interface IState {
      loading: boolean;
-@@ -460,6 +461,12 @@ export default class SecureBackupPanel extends React.PureComponent<{}, IState> {
+@@ -244,25 +245,27 @@ export default class SecureBackupPanel extends React.PureComponent<{}, IState> {
+             if (MatrixClientPeg.get().getKeyBackupEnabled()) {
+                 statusDescription = <p>✅ {_t("This session is backing up your keys. ")}</p>;
+             } else {
+-                statusDescription = (
```

Comment:
I would rather keep this text for the beta phase to be able to understand easily what is going on.